newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -157,6 +157,8 @@ func TestAccNewRelicNrqlAlertCondition_NerdGraphBaseline(t *testing.T) {
 				// we don't set deprecated fields on import
 				ImportStateVerifyIgnore: []string{
 					"term",
+					"critical",
+					"warning",
```

Comment:
Fairly certain we shouldn't ignore these fields on import. I think we also need to make a test that uses those attributes (if we haven't already). Not sure how big of a lift it would be to set the new attributes on import instead of `term`.

Reply:
I can dig a bit on that, shouldn't be too complex.